xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/drawer.less

```diff
@@ -96,7 +96,7 @@
       }
 
       // Avatar styling
-      img {
+      img.avatar {
```

Comment:
The `avatar` class is only applied for the default avatar, but not when an actual avatar image is uploaded (https://jira.xwiki.org/browse/XWIKI-21743)